internal/util/httpserver.go

```diff
@@ -24,3 +26,20 @@ func StartMetricsServer(c *Config) {
 		FatalLogMsg("failed to listen on address %v: %s", addr, err)
 	}
 }
+
+// EnableProfiling enables golang profiling.
+func EnableProfiling() {
+	for _, profile := range runtime_pprof.Profiles() {
+		name := profile.Name()
+		handler := pprof.Handler(name)
+
+		http.Handle(name, handler)
```

Comment:
It would help if you can list the name/path where each handler can be reached. Anyone that has the profiling enabled will then be able to see what this particular executable supports (and new Golang versions might add/drop pprof profiles).

Reply:
okay